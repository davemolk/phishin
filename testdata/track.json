{"success":true,"total_entries":1,"total_pages":1,"page":1,"data":{"id":6693,"show_id":323,"show_date":"1993-04-09","venue_name":"State Theatre","venue_location":"Minneapolis, MN","title":"Stash","position":4,"duration":675971,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":1,"slug":"stash","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null},{"id":4,"name":"Jamcharts","priority":4,"group":"Curated Selections","color":"#888888","notes":"Several minutes of growly, percussive, dissonant, and atypical jamming.","transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/006/693/6693.mp3","waveform_image":"https://phish.in/audio/000/006/693/waveform-6693.png","song_ids":[728],"updated_at":"2023-10-27T22:31:08Z"}}