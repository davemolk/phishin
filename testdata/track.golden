ID:   Date:       Venue:         Location:        Title:  Duration  Set    Mp3
6693  1993-04-09  State Theatre  Minneapolis, MN  Stash   11m 15s   Set 1  https://phish.in/audio/000/006/693/6693.mp3

Tags
Name:      Group:              Notes:
SBD        Audio               
Jamcharts  Curated Selections  Several minutes of growly, percussive, dissonant, and atypical jamming.
